src/components/Card/CardPrimary/CardPrimary.tsx

```diff
@@ -0,0 +1,117 @@
+import styled from "styled-components";
+import { Title } from "../../Typography/Title/Title";
+import { Text } from "../../Typography/Text/Text";
+import { IconName } from "@/components/Icon/types";
+import { Button, Icon, Spacer } from "../..";
+
+export interface CardProps {
+  title: string;
+  image: IconName;
+  hasShadow?: boolean;
+  disabled?: boolean;
+  description: string;
+  infoUrl: string;
+  infoText: string;
+  size?: "sm" | "md";
+}
+
+const Wrapper = styled.div<Pick<CardProps, "size" | "hasShadow" | "size" | "disabled">>`
+  background-color: ${({  theme }) => theme.click.card.primary.color.background.default};
+  border-radius: ${({ theme }) => theme.click.card.primary.radii.all};
+  border: ${({ theme }) => `1px solid ${theme.click.card.primary.color.stroke.default}`};
+  display: flex;
+  max-width: 100%;
+  text-align: center;
+  flex-direction: column;
+  padding: ${({ size = "md", theme }) => theme.click.card.primary.space[size].x} ${({ size = "md", theme }) => theme.click.card.primary.space[size].y};
```

Comment:
this can be 

```js
padding: ${({ size = "md", theme }) =>
    `${theme.click.card.primary.space[size].x} ${theme.click.card.primary.space[size].y}`};
```